packages/blade/src/components/Input/OTPInput/__tests__/OTPInput.web.test.tsx

```diff
@@ -137,6 +138,28 @@ describe('<OTPInput />', () => {
     expect(allInputs[0]).toHaveFocus();
   });
 
+  it('should change focus to previous tab on backspace and delete keys', async () => {
+    const user = userEvent.setup();
+    const label = 'Enter OTP';
+    const otp = '123456';
+
+    const { getAllByLabelText } = renderWithTheme(
+      // eslint-disable-next-line jsx-a11y/no-autofocus
+      <OTPInput label={label} name="otp" />,
+    );
+
+    const allInputs = getAllByLabelText(/character/);
+    await user.type(allInputs[0], otp);
+    await user.click(allInputs[5]);
+    expect(allInputs[5]).toHaveFocus();
+    await user.keyboard('{Backspace}');
+    expect(allInputs[5]).toHaveValue('');
+    expect(allInputs[4]).toHaveFocus();
+    await user.keyboard('{Delete}');
+    expect(allInputs[4]).toHaveValue('');
+    expect(allInputs[3]).toHaveFocus();
```

Comment:
Shouldn't <kbd>Del</kbd> key delete the character in front of cursor 🤔

Reply:
The input element only has 1 character at a time so deleting or backspacing should remove the character within that input itself, right?

Reply:
I think with <kbd>Del</kbd> it should delete the character in the next input box if cursor is after current number, will need to check some existing otp inputs in the wild. Can do a followup PR if needed.

Reply:
Yeah, lets discuss in the standup. The behaviour might be a bit weird if the next input is empty but the one after that has some value. Also how do we handle focus here? With backspace we move the focus.